DS/DSassignment1/ArrayList1.java

```diff
@@ -0,0 +1,203 @@
+import java.util.Arrays;
+
+public class ArrayList1 {
+	
+	public static int defaultSize = 10;
+	public static int size=0;
```

Comment:
ACTION: Make sure other class members in any class are not static unless required.